README.md

```diff
@@ -21,7 +21,15 @@ and bound-constrained optimization
 This package provides an implementation of four classic algorithms for unconstrained/bound-constrained nonlinear optimization:
 
 - `lbfgs`: an implementation of a limited-memory BFGS line-search method for unconstrained minimization;
+    > Boggs, P. T., & Byrd, R. H. (2019). Adaptive, Limited-Memory BFGS
+    > Algorithms for Unconstrained Optimization. SIAM Journal on
+    > Optimization, 29(2), 1282-1299.
+    > DOI: [10.1137/16M1065100](https://doi.org/10.1137/16M1065100)
```

Comment:
We don't implement that strategy. Our implementation is close to that of https://link.springer.com/article/10.1007/BF01589116 but with a different line search.